docs/reference/glossary.asciidoc

```diff
@@ -61,6 +61,16 @@
   `object`. The mapping also allows you to define (amongst other things)
   how the value for a field should be analyzed.
 
+[[glossary-filter]] filter ::
+
+  A filter is a query. It is a kind of query which does not give a score, so it is 
```

Comment:
Maybe you can join the first two sentences to make it more fluent, e.g. "A filter is a <<glossary-query,scoring queries>> that ..." and then join the second sentence. 
Maybe use something like "produces a score" or "scores documents" instead of "give a score"?